parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -47,6 +47,19 @@ write_files:
          "max-file": "5"
       }
     }
+
+- path: "/etc/systemd/system/docker-health-probe.service"
```

Comment:
Could this systemd job definition somehow be shared between master and agent? Maybe using a `{{ template "docker-health-probe.yaml" }}` to include it?

Just wondering how to avoid duplication as the source of errors, but sometimes it's simpler to copy-and-paste anyway.